docs/developer-guide.md

```diff
@@ -0,0 +1,50 @@
+# Developer's Guide to using Weaver
+
+This is an in-progress guide for how to use Weaver to develop your own Semantic Convention registry or codegen.
+
+TODO - Add more getting started guides.
+
+## JQ - Tips and Tricks
+
+While JQ is extremely powerful, it can be finicky to use.  For example, fighting instances where JQ decides to repate a structure vs. nest a list can take some getting used to.
+
+Using an explorer tool like [DevToolsDaily JQ playground](https://www.devtoolsdaily.com/jq_playground/) can dramatically improve your debugging journey in crafting the `jq` expressions you use in rendering templates.
+
+To do so we recommend following [Creating a JSON ouput for your registry](#creating-a-json-output-for-your-registry) and then copy-paste your registry JSON as input into the JQ playground.
```

Comment:
The method described in #creating-a-json-output-for-your-registry is useful for analyzing the context of a template after JQ processing. However, for the debugging phase of a JQ expression, it is simpler to use the following command: `weaver registry resolve -r <registry>`. For more details on this command `weaver registry resolve --help`.

Reply:
Fixed